src/base/static/libs/maps/mapboxgl-provider.js

```diff
@@ -345,6 +345,31 @@ export default (container, options) => {
       map.fitBounds(bounds, options);
     },
 
+    fitLineStringCoords: (coordinates, options) => {
+      // https://www.mapbox.com/mapbox-gl-js/example/zoomto-linestring
+      map.fitBounds(
+        coordinates.reduce(
+          (bounds, coord) => bounds.extend(coord),
+          this._map.makeLngLatBounds(coordinates[0], coordinates[0]),
+        ),
+        options,
+      );
+    },
+
+    fitPolygonalCoords: (coordinates, options) => {
+      // https://www.mapbox.com/mapbox-gl-js/example/zoomto-linestring
+      map.fitBounds(
+        coordinates[0].reduce(
+          (bounds, coord) => bounds.extend(coord),
+          this.mapView.map.makeLngLatBounds(
+            coordinates[0][0],
+            coordinates[0][0],
+          ),
+        ),
+        options,
+      );
+    },
+
     hasLayer: layerId => {
       return !!map.getLayer(layerId);
     },
```

Comment:
The `addMapboxStyle` function is undefined,  and should be caught by your linter.

Reply:
Also, `createRasterTileLayer`, `createWMSLayer`, etc, should be private methods, prefixed with `_`, instead of standalone functions.